index.js

```diff
@@ -520,4 +520,19 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+        {
+                name: 'Quynh Tran',
+                title: 'Student @ DePauw University',
+                hobbies: 'cats, K-pop, travel',
+                avatar:
+                        'https://drive.google.com/file/d/1aEGm3N0dFqTYSbEty1ZSVpetgDAgqzlI/view',
```

Comment:
let's use this URL for reliability: `https://res.cloudinary.com/khoa165/image/upload/v1711671347/viettech/quynh.jpg`